### DISCUSSION TITLE
so what's the problem ????

### DISCUSSION THREAD

**Discussion by Eng-Ahmed-Mahran** [dorssel/usbipd-win#302] [General]
sudo apt-get install usbip
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following NEW packages will be installed:
  usbip
0 upgraded, 1 newly installed, 0 to remove and 198 not upgraded.
Need to get 1,053 kB of archives.
After this operation, 1,178 kB of additional disk space will be used.
Get:1 https://kali.itsec.am/kali kali-rolling/main amd64 usbip amd64 2.0+5.10.28-1kali1 [1,053 kB]
Ign:1 https://kali.itsec.am/kali kali-rolling/main amd64 usbip amd64 2.0+5.10.28-1kali1                                           
Get:1 https://kali.itsec.am/kali kali-rolling/main amd64 usbip amd64 2.0+5.10.28-1kali1 [1,053 kB]                                
Ign:1 https://kali.itsec.am/kali kali-rolling/main amd64 usbip amd64 2.0+5.10.28-1kali1                                           
Get:1 https://kali.itsec.am/kali kali-rolling/main amd64 usbip amd64 2.0+5.10.28-1kali1 [1,053 kB]                                
Ign:1 https://kali.itsec.am/kali kali-rolling/main amd64 usbip amd64 2.0+5.10.28-1kali1                                           
Get:1 https://kali.itsec.am/kali kali-rolling/main amd64 usbip amd64 2.0+5.10.28-1kali1 [1,053 kB]                                
Err:1 https://kali.itsec.am/kali kali-rolling/main amd64 usbip amd64 2.0+5.10.28-1kali1                                           
  Error reading from server - read (5: Input/output error) [IP: 31.47.196.154 443]
E: Failed to fetch https://kali.itsec.am/kali/pool/main/l/linux/usbip_2.0+5.10.28-1kali1_amd64.deb  Error reading from server - re.47.196.154 443]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?

**Comment by dorssel** [comment]
Please read the very content you just posted. It literally tells you "what the problem is". This is not an `usbipd-win` issue. Besides that, please update the title to something more constructive.

Converting to discussion.

**Reply by Eng-Ahmed-Mahran** [reply]
Iam sorry 
Iam new in the community
Sir plz do u know where should I go to fix the problem

**Reply by dorssel** [reply]
Ultimately, at Kali: https://www.kali.org/community/

But you could first try to resolve the 198 updates you are behind:
```
sudo apt update
sudo apt upgrade
sudo apt install usbip
```
